macros/generic_tests/test_unique_combination_of_columns.sql

```diff
@@ -77,7 +90,11 @@
         {% endif %}
     {% endif %}
 {% else %}
-    select 'dummy_value' as "dummy"
+    {%- if target.type == 'databricks' -%}
+        select '"dummy_value"' as `dummy`
```

Comment:
In this dummy query, we do not need the double quotes

Reply:
Sure, it also doesn't break anything, but I'll remove them.